airbyte-webapp/src/config/defaultConfig.ts

```diff
@@ -20,7 +20,7 @@ const defaultConfig: Config = {
   segment: { enabled: true, token: "" },
   healthCheckInterval: 20000,
   version: "dev",
-  apiUrl: `${window.location.protocol}//${window.location.hostname}:8001/api/v1/`,
```

Comment:
Development change left over. To only overwrite this in development, just create a file called `.env.development` in `airbyte-webapp` and put the following into it:

```
REACT_APP_API_URL=https://...
```